hackathon_site/review/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from event.models import User
+from registration.models import Application
+
+
+class Review(models.Model):
+    STATUS_CHOICES = [
+        ("Accepted", "Accepted"),
+        ("Waitlisted", "Waitlisted"),
+        ("Rejected", "Rejected"),
+    ]
+
+    reviewer = models.ForeignKey(
+        User, related_name="reviews", on_delete=models.SET_NULL, null=True
+    )
+    application = models.OneToOneField(
+        Application, on_delete=models.SET_NULL, null=True
+    )
+    interest = models.IntegerField(default=False, null=False)
```

Comment:
Add a `MinValueValidator` and `MaxValueValidator` to these 3 fields limiting options from 0 or 1 to 10 perhaps? Or should we leave it open

https://docs.djangoproject.com/en/3.1/ref/validators/#django.core.validators.MinValueValidator

Reply:
See overview at the top